### ISSUE TITLE
`bugsnag-source-maps upload-react-native` is timing out

### ISSUE THREAD

**Issue by TomSwift** [bugsnag/bugsnag-source-maps#65]
This started failing recently for us.   Is this a service issue or did something change?   Using the latest 2.1.0 version.

```
$ /Users/nicholashodapp/Documents/Projects/iOS/Narwhal/node_modules/.bin/bugsnag-source-maps upload-react-native --api-key ***redacted*** --code-bundle-id com.remitly.remitly --platform ios --source-map ./build/ios/index.ios.bundle.map --bundle ./build/ios/index.ios.bundle
ℹ Preparing upload of React Native source map (release / ios)                       13:39:25
› Reading source map "./build/ios/index.ios.bundle.map"                             13:39:25
› Reading bundle file "./build/ios/index.ios.bundle"                                13:39:26
ℹ Applying transformations to source map                                            13:39:26
› Ensuring sourcesContent field is populated                                        13:39:26
› Stripping project root from sources                                               13:39:26
› Initiating upload to "https://upload.bugsnag.com/react-native-source-map"         13:39:26

 ERROR  The request timed out.                                                      13:44:32

 Connection timed out

  at ClientRequest.<anonymous> (node_modules/@bugsnag/source-maps/dist/Request.js:190:21)
  at Object.onceWrapper (events.js:421:28)
  at ClientRequest.emit (events.js:315:20)
  at TLSSocket.emitRequestTimeout (_http_client.js:709:9)
  at Object.onceWrapper (events.js:421:28)
  at TLSSocket.emit (events.js:327:22)
  at TLSSocket.Socket._onTimeout (net.js:482:8)
  at listOnTimeout (internal/timers.js:549:17)
  at processTimers (internal/timers.js:492:7)

error Command failed with exit code 1.
```

**Comment by mattdyoung** [comment]
@TomSwift
No known service issues at this time. Could you contact support@bugsnag.com with details of which Bugsnag organization account and project this is for and UTC timestamps of when you've seen failures so we can investigate on our side. Thanks!

**Comment by Johan-dutoit** [comment]
FWIW Also experiencing random timeouts.

**Comment by johnkiely1** [comment]
Just wanted to follow up to say we have now released a new version ([v2.3.0](https://github.com/bugsnag/bugsnag-source-maps/releases/tag/v2.3.0)) of our bugsnag-source-maps tool which will increase the default timeout to 10 mins along within adding an `idle-timeout` config option to give configurability around setting your own timeouts. We have added some documentation on that [here](https://docs.bugsnag.com/build-integrations/js/source-maps-react-native/#option-reference). This should hopefully help with the timeout issues you are experiencing.
